Code/Framework/AzFramework/Tests/DocumentPropertyEditor/DocumentPropertyEditorFixture.cpp

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Tests/DocumentPropertyEditor/DocumentPropertyEditorFixture.h>
+#include <AzCore/Name/NameDictionary.h>
+
+namespace AZ::DocumentPropertyEditor::Tests
+{
+    void DocumentPropertyEditorTestFixture::SetUp()
+    {
+        UnitTest::AllocatorsFixture::SetUp();
+        NameDictionary::Create();
+        AZ::AllocatorInstance<Dom::ValueAllocator>::Create();
```

Comment:
question: Just trying to understand this. Why does this need to be explicitly created?

Reply:
This is a version of the high performance heap allocator used by the generic DOM that doesn't come up with the default allocators fixture, so tests that use the generic DOM need this.